scripts/hid_configurator/README.rst

```diff
@@ -44,12 +45,15 @@ On Windows:
 On Linux:
 
 	pip3 install --user -r requirements.txt
+        sudo apt install libhidapi-hidraw0
 
 This installs:
 
-* HIDAPI library - on Windows provided by `hidapi.dll` or `hidapi-x64.dll`
+* HIDAPI library - on Windows provided by `hidapi.dll` or `hidapi-x64.dll`, on Linux provided by `libhidapi-hidraw0`.
 * pyhidapi Python wrapper - installed from GitHub repository with additional patches included. The version on PyPI is obsolete and will not work.
 
+When using configuration channel for BLE devices on Linux, it recommended to use BlueZ version >= 5.44. In previous versions, the HID device attached by BlueZ may obtain wrong VID and PID values (ignoring values in Device Information Service), which will stop HIDAPI from opening the device.
```

Comment:
Split sentences into separate lanes?

Reply:
Not sure how this should be done in docs. Just checked and there is no need to keep 80 char limit, So I will just put sentences in separate lines.

Reply:
fixed